src/components/Tutorial.tsx

```
@@ -19,89 +23,57 @@ export function Tutorial({ active = 'illustration' }: Props) {
           </a>
         </div>
 
         {/* はじめに（共通） */}
         <div className="rounded-xl border border-gray-200 bg-white/80 p-5 shadow-sm mb-6">
-          <h2 className="text-2xl font-bold text-gray-800 mb-4">はじめに</h2>
-          <p className="text-gray-700 leading-relaxed mb-4">
-            AIニケちゃんは、比較的ガイドラインが緩いキャラクターです。<br />
-            このページでは、AIを使ったニケちゃんの画像 および 動画を生成する方法をご紹介します。
-          </p>
+          <h2 className="text-2xl font-bold text-gray-800 mb-4">{t('tutorial:intro.heading')}</h2>
+          <p className="text-gray-700 leading-relaxed mb-4" dangerouslySetInnerHTML={{ __html: t('tutorial:intro.description') }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)